runtime/libpgmath/lib/common/acos/fma3/sdacos.cpp

```diff
@@ -8,7 +8,7 @@
 
 #if defined(TARGET_LINUX_POWER)
 #include "xmm2altivec.h"
-#elif defined(TARGET_LINUX_ARM64)
+#elif defined(TARGET_ARM64)
```

Comment:
Is `TARGET_ARM64` defined when building on Linux?

Reply:
It is currently not, good catch thanks. It should be defined IMHO. Any thoughts on that?

Reply:
Thanks.